src/NuGet.Core/NuGet.Build.Tasks.Console/NuGet.Build.Tasks.Console.csproj

```diff
@@ -0,0 +1,55 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <TargetFrameworks Condition=" '$(IsVsixBuild)' != 'true' ">$(TargetFrameworksExe)</TargetFrameworks>
+    <TargetFramework Condition=" '$(IsVsixBuild)' == 'true' ">$(NETFXTargetFramework)</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <Shipping>true</Shipping>
+    <PackProject>true</PackProject>
+    <SkipShared>true</SkipShared>
+    <Description>NuGet restore console application.</Description>
+    <!-- NU5100: Suppress warning about a DLL not in a lib folder since on .NET Core the EXE is technically a DLL -->
+    <NoWarn>$(NoWarn);CS1591;NU5100</NoWarn>
+    <XPLATProject>true</XPLATProject>
+    <EnableDefaultNoneItems>false</EnableDefaultNoneItems>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
+    <BuildOutputTargetFolder>contentFiles\any</BuildOutputTargetFolder>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\NuGet.Build.Tasks\NuGet.Build.Tasks.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildPackageVersion)" ExcludeAssets="Runtime" />
+    <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildPackageVersion)" ExcludeAssets="Runtime" />
+    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildPackageVersion)" ExcludeAssets="Runtime" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildPackageVersion)" ExcludeAssets="Runtime" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == '$(NETFXTargetFramework)' ">
+    <Reference Include="Microsoft.Build.Utilities.v4.0" Aliases="MicrosoftBuildUtilitiesv4" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="App.config" Condition=" '$(TargetFramework)' == '$(NETFXTargetFramework)' " />
+  </ItemGroup>
+
+  <ItemGroup Condition="$(DefineConstants.Contains(SIGNED_BUILD))">
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+      <_Parameter1>NuGet.Build.Tasks.Test, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9</_Parameter1>
```

Comment:
The project should get its own test assembly, not reuse an old one.

Reply:
I'd like to fold the assemblies together in the next release personally so I reused the test assembly.  The two share so much logic I thought it made sense.  Its only a separate app because I wasn't allowed to rename the task DLL to an EXE.

Reply:
I think I'd favor a new assembly, but I'm curious what the rest of the team thinks @NuGet/nuget-client. 

Let's not block this PR on this. I will ping them in teams.

Reply:
My 2c. I do generally prefer a 1:1 mapping between production and test projects. This makes so when someone is modifying/reading the production code, it's easy to find the test code. Just look for the test project with the same prefix as the prod project's name.

If the two projects share logic, I hope that means it's in one place and referenced from the other, not duplicated. In which case it can be tested in the one place it's written. I didn't review this PR in detail though, so perhaps I'm missing something.

Reply:
I prefer a separate test assembly for testing code that lives in `NuGet.Build.Tasks.Console`. That makes easy to navigate between test and production code, as @zivkan pointed out.

Also, we can address this in a later PR.

Reply:
For the folding assemblies case, we have tests for `NuGet.CommandLine` (nuget.exe) and for other library assemblies as well (`NuGet.Commands`, `NuGet.Protocol`, etc.)

Reply:
Okay I'll make a new test assembly as a follow-up PR